pallets/fungible/src/lib.rs

```diff
@@ -459,7 +470,9 @@ impl<T: Config> Pallet<T> {
 		));
 	}
 
-	/// Sets the amount of owner tokens that the spender can manage.
+	/// Set allowance for the spender to `transfer` or `burn` owner's tokens.
+	///
+	/// - `amount`: Amount of tokens the spender is allowed to `transfer` or `burn`.
```

Comment:
Would be good to add argument descriptions for the other fields here, too, for consistency - also, `spender`, for example, is not that obvious.